sklearn/linear_model/bayes.py

```diff
@@ -347,7 +347,9 @@ def fit(self, X, y):
         self : returns an instance of self.
         """
         X, y = check_X_y(X, y, dtype=np.float64, y_numeric=True)
-
```

Comment:
I think there is another standard error message when providing a single class. can you please check for it?